sdk/storage/Azure.Storage.Blobs/src/AppendBlobClient.cs

```diff
@@ -410,22 +410,40 @@ public new AppendBlobClient WithSnapshot(string snapshot)
             bool async,
             CancellationToken cancellationToken)
         {
-            var conditions = new AppendBlobRequestConditions { IfNoneMatch = new ETag(Constants.Wildcard) };
-            try
-            {
-                return await CreateInternal(
-                    httpHeaders,
-                    metadata,
-                    conditions,
-                    async,
-                    cancellationToken,
-                    Constants.Blob.Append.CreateIfNotExistsOperationName)
-                    .ConfigureAwait(false);
-            }
-            catch (RequestFailedException storageRequestFailedException)
-            when (storageRequestFailedException.ErrorCode == Constants.Blob.AlreadyExists)
+            using (Pipeline.BeginLoggingScope(nameof(AppendBlobClient)))
             {
-                return default;
+                Pipeline.LogMethodEnter(
+                    nameof(AppendBlobClient),
+                    message:
+                    $"{nameof(Uri)}: {Uri}");
```

Comment:
Any reason to not match the message format used in `CreateInternal`?
It also includes the `BlobHttpHeaders`